wolfssl/wolfcrypt/asn.h

```diff
@@ -140,7 +140,8 @@ enum DN_Tags {
     /* pilot attribute types
      * OID values of 0.9.2342.19200300.100.1.* */
     ASN_USER_ID          = 0x01, /* UID */
-    ASN_DOMAIN_COMPONENT = 0x19  /* DC */
+    ASN_DOMAIN_COMPONENT = 0x19, /* DC */
+    ASN_FAVOURITE_DRINK  = 0x05  /* favouriteDrink */
```

Comment:
Let's keep this enum sorted by ascending value - please add ASN_FAVOURITE_DRINK above ASN_DOMAIN_COMPONENT.

Reply:
Thanks. Fixed